src/crawlee/_types.py

```diff
@@ -46,6 +46,20 @@
 
 RequestTransformAction: TypeAlias = Literal['skip', 'unchanged']
 
+EnqueueStrategy: TypeAlias = Literal['all', 'same_domain', 'same_hostname', 'same_origin']
```

Comment:
This could break if you run the new version with storage populated by the previous version - the `enqueue_strategy` is stored in the request queue and the old version has dashes instead of underscores.

Reply:
I don't have a strong opinion here, but it's probably safer

Reply:
getting back to dashes then